partiql-planner/src/testFixtures/resources/tests/plan-typer/plan-typer-tests-ported.ion

```diff
@@ -0,0 +1,6717 @@
+
+test::{
+  name:"Pets should not be accessible #1",
+  type:"type",
+  body:{
+    statement:"SELECT * FROM pets",
+    session:{
+      catalog:"pql",
+      cwd:[
+      ]
+    },
+    status:FAILURE,
+    assertProblemExists:(
+    ERROR
+      '''Variable pets does not exist in the database environment and is not an attribute of the following in-scope variables [].'''
+    )
+  }
+}
+test::{
+  name:"Pets should not be accessible #2",
+  type:"type",
+  body:{
```

Comment:
Could we flatten the typing logic by moving the fields out of the `body` struct?

Reply:
We can't. The `type` here tells which `TestBuilder` that the `TestBuilderFactory` should return. Then, the `body` struct is passed to the `TestBuilder` to deserialize into a `Test` implementation.